Detector/DetCEPCv4/compact/Beampipe_o1_v01_01.xml

```diff
@@ -15,16 +15,16 @@
   
   <!-- central tube -->
   <section type ="Center"                
-	   start="0*mm"                                  end="TUBE_innerCylinder_zEnd"
+	   start="0*mm"                                  end="500*mm"
```

Comment:
Could you use a variable for this `500*mm`? I found in Line 25, you also hardcode `500*mm`.

Reply:
sure, I have updated in commit.